platformio/util.py

```diff
@@ -4,7 +4,7 @@
 import json
 from os import name as os_name
 from os import getcwd, getenv, listdir, makedirs, utime
-from os.path import dirname, expanduser, isdir, isfile, join, realpath
+from os.path import dirname, expanduser, isdir, isfile, join, realpath, abspath
```

Comment:
PlatformIO uses [isort](https://github.com/timothycrosley/isort) for python imports. Please move `abspath` before `dirname` (alphabetic order)